api/dao/liststorage.py

```diff
@@ -290,7 +291,8 @@ def create_job_and_analysis(self, cont_name, cid, analysis, job, origin):
         gear_name = job['gear']
 
         destination = create_containerreference_from_dictionary({'type': 'analysis', 'id': analysis['_id']})
-        job = Job(gear_name, inputs, destination=destination, tags=tags, config_=job.get('config'), origin=origin)
+        gear_id = str(get_gear_by_name(gear_name)['_id'])
```

Comment:
Why don't we require a gear id here yet? Is this a midstep until all clients support gear versioning?